Sources/Request/Request.swift

```diff
@@ -4,16 +4,20 @@ public struct Request {
 
      public let method: String
      public let path: String
+     public let httpVersion: String
      public var queries: [String: String] 
      public var body: [String: String] 
      public var cookie: String  
+     public var headers: [String: String]
 
-     public init(method: String, path: String, queries: [String: String] = ["fname": "World"], body: [String: String] = [String(): String()], cookie: String = String()) {
+     public init(method: String, path: String, httpVersion: String, queries: [String: String] = ["fname": "World"], body: [String: String] = [:], cookie: String = String(), headers: [String: String] = [:]) {
```

Comment:
Why is defaulting the queries to always include fname as "World" still necessary here? Shouldn't the route be able to handle what happens if no queries are specified?  `queries: [String: String] = ["fname": "World"]`